src/clusterfuzz/_internal/bot/tasks/utasks/analyze_task.py

```diff
@@ -437,6 +441,9 @@ def handle_build_setup_error(output):
         output.uworker_input.job_type,
         wait_time=testcase_fail_wait)
     return
+  testcase_upload_metadata = data_types.TestcaseUploadMetadata.query(
+      data_types.TestcaseUploadMetadata.testcase_id == int(
+          output.uworker_input.testcase_id)).get()
```

Comment:
This could be extracted to a helper, there are 3 occurrences of this code in the file.